dex/src/state.rs

```diff
@@ -1346,15 +1354,15 @@ fn send_from_vault<'a, 'b: 'a>(
     Ok(())
 }
 
-pub(crate) mod account_parser {
+pub mod account_parser {
```

Comment:
This is needed for the fuzz tests, which need to create a `SignerAccount`. Maybe there's an easy way to make this conditional based on feature = "fuzz" ?

Reply:
I believe the trick is:
```
#[cfg(...)]
pub mod exported_stuff {
    pub use super::foo::bar;
}
```
Here's a working playground example: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=2b74df110bbfc00aea5620966ddc032f